.github/CODEOWNERS

```diff
@@ -1,98 +1,2 @@
-
-/SPECS/openssl/* @microsoft/cbl-mariner-openssl
-/SPECS/SymCrypt-OpenSSL/* @microsoft/cbl-mariner-openssl
-/SPECS/SymCrypt/* @microsoft/cbl-mariner-openssl
-
-/SPECS/dnf/* @microsoft/cbl-mariner-package-managers
-/SPECS/dnf-plugins-core/* @microsoft/cbl-mariner-package-managers
-/SPECS/rpm/* @microsoft/cbl-mariner-package-managers
-/SPECS/tdnf/* @microsoft/cbl-mariner-package-managers
-
-/SPECS/containerd/* @microsoft/cbl-mariner-container-runtime
-/SPECS/docker-buildx/* @microsoft/cbl-mariner-container-runtime
-/SPECS/docker-cli/* @microsoft/cbl-mariner-container-runtime
-/SPECS/kata-containers/* @microsoft/cbl-mariner-kata-containers
-/SPECS/kata-containers-cc/* @microsoft/cbl-mariner-kata-containers
-/SPECS/moby-containerd-cc/* @microsoft/cbl-mariner-kata-containers
-/SPECS/moby-engine/* @microsoft/cbl-mariner-container-runtime
-/SPECS/runc/* @microsoft/cbl-mariner-container-runtime
-/SPECS/virtiofsd/* @microsoft/cbl-mariner-kata-containers
-
-/SPECS/cloud-hypervisor-cvm/* @microsoft/cbl-mariner-kata-containers
-
-/SPECS/cloud-init/* @microsoft/cbl-mariner-provisioning
-/SPECS/walinuxagent/* @microsoft/cbl-mariner-provisioning
-
-# Modifications to the toolkit requires reviews from the toolkit team
-/toolkit/ @microsoft/cbl-mariner-tooling
-
-# Docs to be reviewed by general Azure Linux devs
-/toolkit/docs/ @microsoft/cbl-mariner-devs
-
-# Default image configurations to be reviewed by general Azure Linux devs
-/toolkit/imageconfigs/ @microsoft/cbl-mariner-devs
-
-# Package and toolchain manifests to be reviewed by toolchain team
-/toolkit/resources/manifests/package/pkggen_core_aarch64.txt @microsoft/cbl-mariner-toolchain
-/toolkit/resources/manifests/package/pkggen_core_x86_64.txt @microsoft/cbl-mariner-toolchain
-/toolkit/resources/manifests/package/toolchain_aarch64.txt @microsoft/cbl-mariner-toolchain
-/toolkit/resources/manifests/package/toolchain_x86_64.txt @microsoft/cbl-mariner-toolchain
-
-# Modifications to the raw toolchain require admin approval.
-/toolkit/scripts/toolchain/container/* @microsoft/cbl-mariner-admins
-/toolkit/scripts/toolchain/cgmanifest.json @microsoft/cbl-mariner-admins
-/toolkit/scripts/toolchain/create_toolchain_in_container.sh @microsoft/cbl-mariner-admins
-
-# Modifications to the trusted CA certificates require admin approval.
-/SPECS/*ca-certificates*/* @microsoft/cbl-mariner-admins
-
-# Image Customizer
-/toolkit/tools/imagecustomizer/ @microsoft/cbl-mariner-imagecustomizer
-/toolkit/tools/imagecustomizerapi/ @microsoft/cbl-mariner-imagecustomizer
-/toolkit/tools/osmodifier @microsoft/cbl-mariner-imagecustomizer
-/toolkit/tools/pkg/imagecustomizerlib/ @microsoft/cbl-mariner-imagecustomizer
-/toolkit/tools/pkg/osmodifierlib @microsoft/cbl-mariner-imagecustomizer
-/toolkit/scripts/build_tag_imagecustomizer.mk @microsoft/cbl-mariner-imagecustomizer
+# For stable release branches, ensure stable release maintainers are added as code reviewers
+* @microsoft/cbl-mariner-stable-maintainers
```

Comment:
Why do we want that in 3.0? And why remove code owners of other areas? We're going to miss all the notifications for the code and packages we own, while at the same time the maintainers will most probably get overwhelmed with PRs to review. We already have severe review delays in 2.0, so I'm a bit worried about this change.

Reply:
Discussed offline - summary is we're going to start with confining the stable-maintainer reviewers to the SPECS folder first, and select portions of the toolkit (specifically docs, manifests, imageconfig definitions)